### ISSUE TITLE
Loss and MSE fluctuate too much and are unstable during training.

### ISSUE THREAD

**Issue by ZhuYi2000** [JanEGerken/HEAL-SWIN#8]
Hi,

First of all, I want to thank you for your incredible work on this project. It's been immensely helpful and insightful.

However, I've encountered an issue with the training process where both the loss and MSE fluctuate significantly, as shown in the attached graph. Despite initial progress, the instability in the validation loss raises concerns about potential overfitting.

Could you please provide some guidance or suggestions on how to address this? I would greatly appreciate your insights.
![mse_curve_hp](https://github.com/user-attachments/assets/1e213590-a609-4ab4-a558-9f42723deadd)
![loss_curve_hp](https://github.com/user-attachments/assets/2f7e5748-700e-494e-adff-fe761164b0c0)

**Comment by ZhuYi2000** [comment]
The command is: 
python3 run.py --env singularity train --config_path=heal_swin/run_configs/depth_estimation/depth_swin_hp_train_run_config.py
and i didn't change any config in depth_swin_hp_train_run_config.py

**Comment by JanEGerken** [comment]
We’ve seen similar behavior to this. You could try to activate gradient clipping by setting the `gradient_clip_val` in the `PLConfig` dataclass in the configuration file.
